.venv/bin/Get-GPPPassword.py

```
@@ -0,0 +1,321 @@
+                                "tagName": xmltype,
+                                "attributes": [
+                                    ("newName", read_or_empty(property, "newName")),
+                                    ("userName", read_or_empty(property, "userName")),
+                                    ("cpassword", read_or_empty(property, "cpassword")),
+                                    ("password", self.decrypt_password(read_or_empty(property, "cpassword"))),
+                                    ("changed", read_or_empty(property.parentNode, "changed")),
+                                ],
+                                "file": filename
+                            })
+
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)